src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { findTabById } from '../../utils';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const currentTab = findTabById(tabs, selectedTabId);
```

Comment:
```suggestion
  const selectedTab = findTabById(tabs, selectedTabId);
```

Reply:
It's a better name, because as  here we are rendering list of tabs, `currentTab` means that this is a tab on some iteration step